frontend/src/components/Items.vue

```diff
@@ -75,25 +78,37 @@ export default {
     isFiltered(itemFqid) {
       return this.filteredItems.has(itemFqid);
     },
+   onClickPanel(contextualItem, index) {
+      this.$emit("clickPanel", contextualItem, index);
+    },
+    getClass(item) {
+      return {
+        readItemStyle: item.readState === "READ",
+        unreadItemStyle: item.readState === "UNREAD" || item.readState === "WAIT"
+      };
+    },
   },
   computed: {
-    contextualItems: function() {
+    contextualItems: function () {
       const result = [];
       for (const item of this.items) {
         if (item.action1) {
-          if (item.action1.contexts) { // multiple contexts -- duplicate item
+          if (item.action1.contexts) {
```

Comment:
try to avoid making formatting changes in unmodified code.
It makes reviewing harder...